test/unit/spec/common/jobsData.js

```diff
@@ -0,0 +1,291 @@
+define(['common/format'], (format) => {
```

Comment:
moved this out of the jobsSpec test file and into its own file so it can be used in other tests